datasets/ronec/README.md

```diff
@@ -0,0 +1,156 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+- found
+languages:
+- ro
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- structure-prediction
+task_ids:
+- named-entity-recognition
+---
+
+# Dataset Card for [Needs More Information]
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/dumitrescustefan/ronec
+- **Repository:** https://github.com/dumitrescustefan/ronec
+- **Paper:** https://arxiv.org/abs/1909.01247
+- **Leaderboard:** [Needs More Information]
+- **Point of Contact:** dumitrescu.stefan@gmail.com, avram.andreimarius@gmail.com
+
+### Dataset Summary
+
+We present RONEC - the Named Entity Corpus for the Romanian language. The corpus contains over 26000 entities in 5000 annotatedsentences, belonging to 16 distinct classes. The sentenceshave been extracted from a copy-right free newspaper, covering severalstyles. This corpus represents the first initiative in the Romanian language space specifically targeted for named entity recognition.
+
+### Supported Tasks and Leaderboards
+
+[Needs More Information]
+
+### Languages
+
+The text dataset is in Romanian (`ro`)
+
+## Dataset Structure
+
+### Data Instances
+
+An example looks like this:
+
+{'end': [19, 28, 42, 80], 
+ 'id': '1', 
+ 'ronec_class': [13, 4, 13, 8], 
+ 'sentence': ['Secretarul de stat al S.U.A., Colin Powell, începe un turneu în țările asiatice afectate de valurile seismice uriașe.'], 
+ 'start': [1, 22, 31, 72]
+}
+
+### Data Fields
+
+- `id`: a `string` feature describing the index of the sentence in the dataset
+- `sentence`: a `list` of `string` features.
```

Comment:
to follow the conll2003 format can you rename it `tokens` instead of `sentence` ?